From 7d2f5c37fd611e488d67d6a231fc567f870a05e6 Mon Sep 17 00:00:00 2001
From: David Menache <dave@doap.com>
Date: Mon, 7 Nov 2011 07:14:12 -0800
Subject: [PATCH 2/2] Commit via Mac

---
 DONTREADME |    7 +++++++
 1 files changed, 7 insertions(+), 0 deletions(-)
 create mode 100644 DONTREADME

diff --git a/DONTREADME b/DONTREADME
new file mode 100644
index 0000000..38d9f1d
--- /dev/null
+++ b/DONTREADME
@@ -0,0 +1,7 @@
+Puffy Clouds and Sunshine Animation
+Live Demo @ http://core.doap.com/
+done with javascript and html5 canvas. 
+by David Menache dave@doap.com
+and 
+Bobby Damery bobby@doap.com
+Visit Doap.com for a inexpensive self-serve CMS hosting solution.
-- 
1.7.7

